app/assets/stylesheets/common.scss

```diff
@@ -542,8 +541,6 @@ body.compact {
   #sidebar {
     float: left;
     width: $sidebarWidth;
-    background: #fff;
```

Comment:
Removing the background colour here means the welcome message for new users has no background and the text is overlaid directly on the map.

Other uses of the sidebar seem to work somehow, despite not having an explicit backgound colour - they seem to inherit the browser default I think but that doesn't happen with the welcome message?

Reply:
OK, I'll investigate and fix this.